### ISSUE TITLE
[Bug]: Browsing is not working

### ISSUE THREAD

**Issue by neubig** [OpenDevin/OpenDevin#1276]
### Is there an existing issue for the same bug?

- [X] I have checked the troubleshooting document at https://github.com/OpenDevin/OpenDevin/blob/main/docs/guides/Troubleshooting.md
- [X] I have checked the existing issues.

### Describe the bug

When I ask a question that requires browsing the web to get the answer, OpenDevin does not use the "browsing" tab.

For instance, I asked
```
Please resolve this pull request: https://github.com/OpenDevin/OpenDevin/issues/1275
```

In trying to resolve the pull request, OpenDevin tried to install Playwright to browse the web instead of using built-in browsing capability, and the browsing tab said "no screenshot available".

### Current Version

```bash
`ghcr.io/opendevin/opendevin:0.3.1`
```

### Installation and Configuration

```bash
export LLM_API_KEY="sk-..."
export WORKSPACE_DIR=$(pwd)/workspace
```

### Reproduction Steps

1. Ask OpenDevin: `Please resolve this pull request: https://github.com/OpenDevin/OpenDevin/issues/1275`

### Logs, Errors, Screenshots, and Additional Context

It seems that this is a relevant error:

```
STEP 2

23:12:20 - opendevin:INFO: agent_controller.py:89
PLAN
Resolve this pull request: https://github.com/OpenDevin/OpenDevin/issues/1275
23:12:26 - opendevin:INFO: agent_controller.py:107
ACTION
BrowseURLAction(url='https://github.com/OpenDevin/OpenDevin/pull/1275', action=<ActionType.BROWSE: 'browse'>)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
23:12:27 - opendevin:INFO: agent_controller.py:160
OBSERVATION
BrowserType.launch: Executable doesn't exist at /root/.cache/ms-playwright/chromium-1112/chrome-linux/chrome
╔════════════════════════════════════════════════════════════╗
║ Looks like Playwright was just installed or updated.       ║
║ Please run the following command to download new browsers: ║
║                                                            ║
║     playwright install                                     ║
║                                                            ║
║ <3 Playwright Team                                         ║
╚════════════════════════════════════════════════════════════╝
```

**Comment by neubig** [comment]
This may be duplicate with https://github.com/OpenDevin/OpenDevin/issues/1172
And solved by https://github.com/OpenDevin/OpenDevin/pull/1184

I'll try to take a look.

**Comment by neubig** [comment]
Should be fixed by #1184
